lib/features/home/presentation/widgets/meal_plan_list.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:flutter/material.dart';
+import 'package:mealmaster/features/home/presentation/widgets/date_list_tile.dart';
+import 'package:mealmaster/features/home/presentation/widgets/recipe_list_tile.dart';
+import 'package:mealmaster/features/recipes/domain/recipe.dart';
+
+class MealPlanList extends StatefulWidget {
+  final List<Recipe> recipes;
+  const MealPlanList({
+    super.key,
+    required this.recipes,
+  });
+
+  @override
+  State<MealPlanList> createState() => _MealPlanListState();
+}
+
+class _MealPlanListState extends State<MealPlanList> {
+  final List<String> _days = ['MO', 'DI', 'MI', 'DO', 'FR', 'SA', 'SO'];
+
+  void onReorder(int oldIndex, int newIndex) {
```

Comment:
Nice das das Reordern schon funktioniert :) Kleiner Hinweis: Die Tage werden noch komisch verschoben. Also sie bleiben nicht in der Reihenfolge wie ich sie gezogen habe.
Können wir uns aber gerne als neues Issue aufschreiben.